WTG.Analyzers/Analyzers/PointlessOverride/PointlessOverrideCodeFixProvider.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+
+namespace WTG.Analyzers
+{
+	[ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(PointlessOverrideCodeFixProvider))]
+	[Shared]
+	class PointlessOverrideCodeFixProvider : CodeFixProvider
+	{
+		public override ImmutableArray<string> FixableDiagnosticIds { get; } = ImmutableArray.Create(
+			Rules.RemovePointlessOverridesDiagnosticID);
+
+		public override FixAllProvider GetFixAllProvider() => PointlessOverrideFixAllProvider.Instance;
+
+		public override Task RegisterCodeFixesAsync(CodeFixContext context)
+		{
+			var diagnostic = context.Diagnostics.First();
+			context.RegisterCodeFix(
+				CodeAction.Create(
+					title: "Remove override.",
```

Comment:
That would require either giving each member type a different id or identifying the target member before the fix is selected, both I would like to avoid.